lightning-invoice/src/lib.rs

```diff
@@ -607,6 +607,20 @@ impl<D: tb::Bool, H: tb::Bool, T: tb::Bool, C: tb::Bool> InvoiceBuilder<D, H, T,
 	}
 }
 
+impl<D: tb::Bool, H: tb::Bool, T: tb::Bool, C: tb::Bool> InvoiceBuilder<D, H, T, C, tb::True> {
+	/// Sets the `basic_mpp` feature as optional.
+	pub fn basic_mpp(mut self) -> Self {
```

Comment:
Should we have a similar function for `var_onion_optin`?

Reply:
Nope. It's only needed because it is a feature dependency of `payment_secret`, so there's no need to set it individually.